packages/compile-solidity/index.js

```diff
@@ -84,10 +84,14 @@ const Compile = {
       "resolver"
     ]);
 
+    // we need to remove all JSON files so Vyper-related stuff doesn't get
+    // sent to the compiler - the Profiler will find JSON imports
+    const filteredPaths = paths.filter(path => path.extname !== ".json");
```

Comment:
This line doesn't look right.  Did you mean to do `path.extname(path)`?  Although, um, I guess that wouldn't work due to one name meaning two things.

Alternatively, you could just check that `path.endsWith(".sol")`.

Reply:
Doh, that is a mistake